omero/sysadmins/server-upgrade.rst

```diff
@@ -19,9 +19,10 @@ the level of detail; upgrading should be a straightforward process.
     instructions for those prior versions because they may describe
     important steps that these instructions assume to already have been
     done by OMERO |previousversion| users. Before proceeding with these
-    instructions you may therefore first need to read the `instructions
+    instructions you may first need to read the `instructions
     <https://docs.openmicroscopy.org/latest/omero5.3/sysadmins/server-upgrade.html>`_
-    for upgrading *to* OMERO |previousversion|.
+    for upgrading *to* OMERO |previousversion| because some extra steps
+    may be required beyond simply running the SQL upgrade scripts.
```

Comment:
Maybe just add "...upgrade scripts described below". Just to clarify that the main upgrade steps, even for <5.3 are included in the upgrade instructions on this page.